CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+
+Please start by discussing the change via [issue](https://github.com/LDSSA/curriculum-development/issues).
+
+# Code of Conduct
+
+LDSSA's [Code of Conduct](https://github.com/LDSSA/wiki/wiki/Code-of-Conduct).
+
+# Teaching Philosophy
+
+LDSSA's [Teaching Philosophy](https://github.com/LDSSA/wiki/wiki/Teaching-Philosophy).
+
+# Pull Request Process
+
+We use the [GitHub flow](https://guides.github.com/introduction/flow/).
+
+1. Create a new branch, following the naming convention `ISSUE_NUMBER/ISSUE_TITLE`
+2. Update the file(s) with the proposed changes
+3. Commit the proposed changes
+4. Open a Pull Request to discuss the changes.
+
+# Approval Process
+
+1. Anyone on `#curriculum` can discuss, review, or upvote/downvote Pull Requests
+2. Reviews by Teaching, Quality Assurance, and Student Success are optional but encouraged
+3. Final approval by the Curriculum Development team (merge to master and close the issue).
+
+For Specializations with a Lead assigned, further changes need to be approved by the Specialization Lead.
+
+# Curriculum Development Team
+
+* João Ascensão (@jtascensao)
+* Pedro Fonseca (@PedroGfonseca)
+
+# Specialization Leads
+
+# Milestones (Batch 3)
+
+**Submit ideas before the 18th of March.**
+
+New discussions after these dates will be considered for the next Batch.
+
+Find the important dates for Curriculum Development below.
+
+| Deadline                                          | Milestone                  | Enabler     |
+|---------------------------------------------------|----------------------------|-------------|
+| 15th March                                        | Call for ideas             | @jtascensao |
+| 18th March                                        | Curriculum first draft     | @jtascensao |
+| 31st March                                        | Close curriculum           | @jtascensao |
+| 6 weeks before the start of each Specialization   | Submit detailed curriculum | Specialization Leads |
```

Comment:
I'm confused here about who has the responsibility to develop the curriculum for the Specialisations: should it be the curriculum team, the specialisation lead, or both?

Then, I think that having the curriculum of the specialisations closed only 6 weeks before the start of the specialisation may be too late, for two reasons:
- The spec. teams may need more than 6 weeks to prepare the learning materials
- There should be an overall picture of the entire curriculum before the start of the academy, to make sure that we're covering everything we find important

Reply:
The main idea is:
1. The Curriculum Team develops a detailed draft of the Specialization's curriculum
2. Specialization Leads are assigned to Specializations, in accordance with the curriculum
3. The curriculum works like "contract" that is binding for the Lead and the Curriculum Team 
3. From the assignment onwards, Leads also need to approve contract changes
3. The Specialization Lead is free to refine the curriculum (if he so desires) until 6 weeks before the start of the Specialization.

What do you think?

Reply:
@jtascensao makes perfect sense to me!